.github/workflows/python-app.yml

```diff
@@ -40,3 +40,81 @@ jobs:
     - name: Test with pytest
       run: |
         pytest
+    - name: Install pypa/build
```

Comment:
Do we want to build on every push? or should we run this only on tag?

Reply:
good catch, only on tag. I'll make the change.

Reply:
actually, keeping this as is. building only takes a few seconds, and requires the same container as our testing, so it makes sense to just get it done on PR (perhaps something in the setup.py changed affecting the ability to build), publishing the build to PyPi only happens on tag/release